modules/apps/object/object-admin-rest-test/src/testIntegration/java/com/liferay/object/admin/rest/resource/v1_0/test/ObjectDefinitionResourceTest.java

```diff
@@ -123,6 +124,36 @@ public void testGetObjectDefinition() throws Exception {
 			objectDefinition.getRestContextPath());
 	}
 
+	@Override
+	@Test
+	public void testGetObjectDefinitionsPage() throws Exception {
+
+		// super.testGetObjectDefinitionsPage();
```

Comment:
Why is this commented?

Reply:
Oh sorry! I commented this because it uses the data from the DB